shared/src/main/scala/amf/client/remod/amfcore/config/RenderOptions.scala

```diff
@@ -2,11 +2,11 @@ package amf.client.remod.amfcore.config
 
 import amf.core.errorhandling.{ErrorHandler, UnhandledErrorHandler}
 import amf.core.metamodel.Field
-
+import amf.core.emitter.{RenderOptions => LegacyRenderOptions}
 /**
   * Immutable implementation of render options
   */
-private[remod] case class RenderOptions(
+private[amf] case class RenderOptions(
```

Comment:
Is the `eh` left for backwards compatibility only? If it is I'd add a big TODO to erase it afterwards.

Reply:
eh is deleted from the RenderOptions defined in remod package. Client and internal (legacy) render options still have the error handler.